src/main/java/com/uber/jenkins/phabricator/conduit/DifferentialClient.java

```diff
@@ -114,8 +114,22 @@ public JSONObject postComment(String message) throws IOException, InterruptedExc
     }
 
     protected JSONObject callConduit(String methodName, Map<String, String> params) throws IOException, InterruptedException, ArcanistUsageException {
-        ArcanistClient arc = new ArcanistClient(this.arcPath, "call-conduit", params, this.conduitToken, methodName);
+        ArcanistClient arc = getArcanistClient(methodName, params);
         return arc.parseConduit(this.launcher.launch(), this.launcher.getStderr());
     }
 
+    /**
+     * Get a new arcanist client.
+     * @param params parameters to pass to arcanist
+     * @return a new ArcanistClient
+     */
+    protected ArcanistClient getArcanistClient(String methodName, Map<String, String> params) {
```

Comment:
I was planning on separating this out to mock the arcanist client but didn't end up doing it. Probably should move this back to the inline thing that it was before, although maybe this is considered better style?